### DISCUSSION TITLE
2.7 - Edit Pod Page Improvements Part2?

### DISCUSSION THREAD

**Discussion by quasel** [pods-framework/pods#6733] [Ideas]
Related #4175

- Add Warning if you want to leave the Edit Pod Page without "Save Pod" ( Like clicking Edit Pods again )
- ~Just clicking edit already colors the Field red - without any changes done the field settings~
- ~maybe add a "save all" to the warning popup?~ 
![popup](https://cloud.githubusercontent.com/assets/9464217/26177185/0360b974-3b59-11e7-91e0-4539168ab518.png)

## Dev tasklist for 2.9

- [ ] Clicking out of the Add Group / Add Field modal without clicking "Cancel" prompts the admin to confirm they want to discard the new group / field
- [ ] Clicking out of the Edit Group / Edit Field modal without clicking "Cancel" prompts the admin to confirm they want to discard the changes
- [ ] Navigating away from the edit pod screen if there were any fields that changed or groups / fields reordered should prompt the admin to confirm they want to discard the changes

**Comment by quasel** [comment]
Btw I really love the changes! Big improvement!

**Comment by JoryHogeveen** [comment]
Sounds good! Don't know if there already is an option to create alert/confirm messages like that in the current codebase. @sc0ttkclark @pglewis Any thoughts?

**Comment by sc0ttkclark** [comment]
@zrothauser Something for 2.9, let's make it so:

* Clicking out of the Add Group / Add Field modal without clicking "Cancel" prompts the admin to confirm they want to discard the new group / field
* Clicking out of the Edit Group / Edit Field modal without clicking "Cancel" prompts the admin to confirm they want to discard the changes
* Navigating away from the edit pod screen if there were any fields that changed or groups / fields reordered should prompt the admin to confirm they want to discard the changes
